include/macros.h

```diff
@@ -166,6 +166,12 @@ extern struct GraphicsContext* __gfxCtx;
 #define OVERLAY_DISP    __gfxCtx->overlay.p
 #define DEBUG_DISP      __gfxCtx->debug.p
 
+#if ENABLE_SPLIT_ZBUFFER
+#define GET_ZBUFFER ((u16*)(SysCfb_GetFbPtr(2)))
```

Comment:
I don't think we should be returning a pointer to the z-buffer in a function that returns framebuffer pointers

I think this should be moved to the spec and do all the necessary address assignments there:
```
beginseg
    name "buffers"
    flags NOLOAD
    include "$(BUILD_DIR)/src/buffers/gfxbuffers.o"
    include "$(BUILD_DIR)/src/buffers/audio_heap.o"
#if !ENABLE_SPLIT_ZBUFFER
    include "$(BUILD_DIR)/src/buffers/zbuffer.o"
#endif
endseg

#if ENABLE_SPLIT_ZBUFFER
beginseg
    name "zbuffer"
    flags NOLOAD
#if ENABLE_SPLIT_FRAMEBUFFERS
    address 0x80500000 /* Leave room for the split framebuffers in 2 banks */
#else
    address 0x80600000 /* Both framebuffers in 1 bank */
#endif
    include "$(BUILD_DIR)/src/buffers/zbuffer.o"
endseg
#endif
```